### ISSUE TITLE
This project seems very interesting..

### ISSUE THREAD

**Issue by metalice** [cezerin/cezerin#562]
hello,

im looking at this project and it seems very i
what about open a slack for quick discussions? 

Matan

**Comment by juancamiloqhz** [comment]
Hello, I'm very interested too. Please open the slack group

**Comment by zgelp** [comment]
Hey, I would like to implement some more features (user accounts and payment gateways), so slack group will be more than welcome.

**Comment by dougkulak** [comment]
https://join.slack.com/t/cezerin-ecommerce/shared_invite/enQtNDMwOTEyMTk0OTk1LTQwNmEzNTgyN2Y5NjZjYzQ3N2FjY2U4ZWQxYTU0M2EwYWRkMTk5OGMzMzVkYTk0ZTEzZDYxZjk5MGQ1MWZmMjQ

**Comment by jdutriaux** [comment]
@dougkulak The link you sent seems dead :(

**Comment by dougkulak** [comment]
Yeah the original owners are MIA I guess. Will probably have to just fork
and promote the fork

**Comment by huanghunz** [comment]
Hi dougkulak, yes please, that will be very helpful!

**Comment by dougkulak** [comment]
Yes we are doing just that! Everyone bookmark CaesarCommerce.com and check
back in a week!!! Visit DougKulak.com and click “Let’s Talk” and tell me if
you want to help contribute, a bit about yourself and your background, and
what your specialties are and what you most see yourself doing to help out
the new Cezerin fork called CaesarCommerce! This is going to be great, we
just need to allow some time for us to set up the tools we will need for
this project to succeed. If anyone has some GitHub repos that they really
admire and think we should replicate and tweak the contribution process of,
please let me know. We have a Slack channel, check it out!

https://join.slack.com/t/caesarcommerce/shared_invite/enQtNDU5OTU4ODIyMzUyLWU2NWRmMzI3MTMxYTlkODFkZTdmZTM4MjdiNWQ2NTg4MzIyNDE1Yzc0ZGY5ZmFkNDc2Njc5Y2NkNWQxNDk4ODM

Thanks!

Doug

**Comment by neunygph** [comment]
Hi @dougkulak 

Any update on moving this project to CaesarCommerce.com ?

Thanks,
